specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -348,6 +348,14 @@ model Test {
   @added(APIVersions.v2024_07_01_preview)
   metricsReferenceIdentityId?: string;
 
+  @doc("Type of the managed identity built in load test engines")
+  @added(APIVersions.v2024_12_01_preview)
+  engineBuiltinIdentityType?: ManagedIdentityType = ManagedIdentityType.SystemAssigned;
+
+  @doc("Resource Id of the managed identity built in to load test engines")
+  @added(APIVersions.v2024_12_01_preview)
+  engineBuiltinIdentityId?: string[];
```

Comment:
Can there be a mix of user assigned and system identities. I assume not, but confirming